romancal/flatfield/flat_field_step.py

```diff
@@ -15,18 +16,20 @@ class FlatFieldStep(RomanStep):
 
     def process(self, input):
 
-        input_model = datamodels.open(input)
-
+        input_model = rdm.open(input)
```

Comment:
It might be wise to start encouraging the use of `Step.open_model` to make these step classes more reusable between missions

Reply:
Good point. Done.  (but not yet pushed since there is a regtest issue I need to look at).